instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/Config.java

```diff
@@ -46,10 +47,17 @@ public static Config get() {
     return INSTANCE;
   }
 
-  public static Config create(Map<String, String> allProperties) {
+  public static Config createFromAlreadyNormalized(Map<String, String> allProperties) {
```

Comment:
A bit obtuse name - maybe create with some javadoc is ok?

Reply:
ya, reworked it, moved `AgentConfigBuilder` to this module (renamed now as `ConfigBuilder`) so that tests can use it